### DISCUSSION TITLE
brew install openjdk@11

### DISCUSSION THREAD

**Discussion by aaartemPetrov** [Homebrew/discussions#3209] [Getting started] [answered]
I try to install jenkins-lts using command:
_brew install jenkins-lts_
But on the loading openjdk@11 step installation stops and freezes(i have great dowloading speed and stable connection and i waited about half an hour).

When i try to simply install openjdk@11 i have the same situation:
![Screenshot 2022-04-25 at 12 32 58](https://user-images.githubusercontent.com/99199498/165062748-6d67dea4-4ab7-4749-8a4b-265fa9380b17.png)

What is the reason of this?

**Comment by theobondaz** [comment]
same problem here with others dependencies, internal server error I think

**Comment by NoaBm96** [comment]
Happens to me with every ghcr.io i'm trying to install

**Comment by DrFlibble** [comment, marked as answer]
GitHub Packages seems to be down: https://www.githubstatus.com/
